### ISSUE TITLE
PlutoBoard.run_fileserver() only serves files in the last created package by PlutoBoard.

### ISSUE THREAD

**Issue by liuyxpp** [UniStuttgart-IKR/PlutoBoard.jl#18]
Say I have setup two packages A and B via PlutoBoard. A is created before B. Then, if I run the notebook in A, `PlutoBoard.run_fileserver()` will only load static files from B.

**Comment by Niels1006** [comment]
That might be because it rather calls the function from B over A, I will look into that. Out of curiosity, why do you want to set up two packages using PlutoBoard with one depending on the other?

**Comment by liuyxpp** [comment]
No, A and B are independent. At first, I setup A for my work. But I lost the original static/index.html during development. I want to learn something from it. And I can not find the content of this index.html in the doc site at that time. So I use PlutoBoard to create another package B so that I can refer to the original index.html. After I setup B, when I rerun the notebook in A, the content reverts back to the original index.html. I check the output of run_fileserver() in the A's notebook, and find it loading the static file from folders of B.

**Comment by Niels1006** [comment]
Ohh, I see. This happens because you precompiled PlutoBoard in package A. When you imported it in package B, PlutoBoard was already precompiled, so it didnt't use the new filepath of package B. 
That's a mistake on PlutoBoard's side I will fix.
To force precompilation until the fix you can delete the folder `~/.julia/compiled/v1.11/PlutoBoard` and precompile with `] precompile` in your package.

> And I can not find the content of this index.html in the doc site at that time

All setup files are located in `setup/` (https://github.com/UniStuttgart-IKR/PlutoBoard.jl/blob/main/setup/static/index.html).
But yes, I should add an information where to find those example files to docs.
